vtkext/private/module/vtkF3DEXRReader.cxx

```diff
@@ -142,6 +228,24 @@ void vtkF3DEXRReader::ExecuteDataWithInformation(vtkDataObject* output, vtkInfor
   }
 }
 
+//------------------------------------------------------------------------------
+/**
+ * Read from memory instead of file
+ */
+void vtkF3DEXRReader::SetMemoryBuffer(const void* buff)
```

Comment:
I'm a bit confused here, we need to do that internally, I'm not sure we need an API for that. wdyt @Meakk ?

Reply:
correct. it is already used buy the USD reader

https://github.com/f3d-app/f3d/blob/2aa9d398a739ffc3beb6d998186e4c94234e0efb/plugins/usd/module/vtkF3DUSDImporter.cxx#L900

Reply:
nice!